fixit/rules/explicit_frozen_dataclass.py

```diff
@@ -0,0 +1,262 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import libcst as cst
+import libcst.matchers as m
+from libcst._nodes.whitespace import SimpleWhitespace
+from libcst.metadata import QualifiedName, QualifiedNameProvider, QualifiedNameSource
+
+from fixit import CstLintRule, InvalidTestCase as Invalid, ValidTestCase as Valid
+
+
+class ExplicitFrozenDataclassRule(CstLintRule):
+    """
+    Encourages the use of frozen dataclass objects by telling users to specify the
+    kwarg.
+
+    Without this lint rule, most users of dataclass won't know to use the kwarg, and
+    may unintentionally end up with mutable objects.
+    """
+
+    ONCALL_SHORTNAME = "instagram_server_framework"
```

Comment:
This line is not needed.

Reply:
Ah, thanks for the catch

Reply:
Regarding > Did pyre complain about the any() is not a function?
It didn't state if it was `any()` that was violating. The full error is below. 
```fixit/rules/explicit_frozen_dataclass.py:245:79 Call error [29]: `typing.Union[typing.Callable(tuple.__iter__)[[], typing.Iterator[Variable[_T_co](covariant)]], typing.Callable(typing.Sequence.__iter__)[[], typing.Iterator[cst._nodes.expression.Arg]]]` is not a function.```